src/Action.ts

```diff
@@ -10,10 +10,10 @@ export class Action {
     let filtered = pullRequests.filter((pr) => pr.user?.login == this.userName)
 
     if (filtered.length > 1){
-      let newest = filtered[0].head.ref
       filtered.splice(0, 1)
       for (let i = 0; i < filtered.length; i++){
-        this.api.createComment(filtered[i].number, `Closing PR as a newer one from ${this.userName} exists at ${newest}`)
+
+        this.api.createComment(filtered[i].number, `Closing PR as it is superceeded by #${filtered[i].number}`)
```

Comment:
I think this would be `newest.number`. Otherwise I think we are just telling each PR to look at itself 🙃  (unless I'm mistaken).

Reply:
Nope you're not mistaken at all 🤦 , great catch! Fixed.